File: modified_scripts/julia/fibonacci/qwen2.5-coder_3b.jl

function fib(n)
    if n <= 0
        return 0
    elseif n == 1
        return 1
    else
        a, b = 0, 1
        for i = 2:n
            a, b = b, a + b
        end
        return b
    end
end

File: modified_scripts/rust/fibonacci/qwen2.5-coder_3b.rs

fn fib(n: u32) -> u32 {
    if n <= 1 { return n; }
    let (mut a, mut b) = (0, 1);
    for _ in 2..=n {
        let next = a + b;
        a = b;
        b = next;
    }
    b
}
